config/php-scoper/paypal.inc.php

```diff
@@ -0,0 +1,30 @@
+<?php
+declare( strict_types=1 );
+
+use Isolated\Symfony\Component\Finder\Finder;
+
+return [
+	'prefix'   => 'Give\Vendor\PayPal',
```

Comment:
What's the resulting namespace, here? It seems like the last "\PayPal" part shouldn't be necessary.

Reply:
Fixed